Sources/SwiftDocCUtilities/Action/Actions/InitAction.swift

```diff
@@ -0,0 +1,156 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2021 Apple Inc. and the Swift project authors
```

Comment:
Super nit
```suggestion
 Copyright (c) 2023 Apple Inc. and the Swift project authors
```

Reply:
Fixed in https://github.com/apple/swift-docc/pull/719/commits/2a993c4e59d8222a3359af61a8b4ef0b3be9bdcf